Source/WebCore/platform/mac/PasteboardWriter.mm

```diff
@@ -81,7 +81,7 @@
         }
 
         // WebURLsWithTitlesPboardType.
-        auto paths = adoptNS([[NSArray alloc] initWithObjects:@[ @[ cocoaURL.absoluteString ] ], @[ urlData->title.stripWhiteSpace() ], nil]);
+        auto paths = adoptNS([[NSArray alloc] initWithObjects:@[ @[ cocoaURL.absoluteString ] ], @[ urlData->title.stripLeadingAndTrailingCharacters(deprecatedIsSpaceOrNewline) ], nil]);
```

Comment:
Could use `StringView` here to save an allocation in the rare case that we do strip characters. Would need to use thr explicit `StringView` function that creates an `NSString` since the convenient automatic conversion is built into the older `String` and not the newer `StringView`.

Reply:
I left a FIXME.